mshots_ctl.sh

```diff
@@ -32,7 +32,7 @@ function startservice {
 	fi
 
 	running=`lsmod | grep vfb | awk '{ print $(1) }'`
-	if [ -z $running ] ; then
+	if [ -z $running ] && [ "$ARCH" != "arm64" ] ; then
```

Comment:
Small change here to avoid the modprobe errors on arm64. This doesn't seem to have any effect on the actual screenshot being generated. 🤷🏽

Reply:
>This doesn't seem to have any effect on the actual screenshot being generated. 🤷🏽

I think that there are no effects because on M1 there is enough video RAM allocated by default.